### DISCUSSION TITLE
KEDA operator not scaling to 0 due to invalid offset with kafka_scalar

### DISCUSSION THREAD

**Discussion by kevinmat** [kedacore/keda#1584] [Q&A / Need Help]
Hi,
I am using Scaled object with Kafka triggers on multiple topics. I have found that even if 1 topic in the list of Scalars has no committed offsets , the scaling down to 0 does not occur.
Is this correct observation?

The error in logs are like below:

2021-02-06T07:06:00.633Z	INFO	kafka_scaler	invalid offset found for topic

Thanks,
Kevin

**Comment by ganesh-kr** [comment]
@kevinmat 
Would you mind sharing scaled object configuration for multiple topics of kafka type. We have similar requirement. 
It would be helpful for us
